.travis.yml

```diff
@@ -44,16 +44,22 @@ env:
     - secure: "KDP1zPlruadQIbkvXHWHx6rLcTJ3P7AK2Io7IBnAiW3QETDxVksThXKY2Hey25fXw8+UV0nXuQ4v4gMNmghjvCU1k3N/vkIk9AUDC19z1IM282GumvJjZ+6WdHGvMn1sRPi3Jt3ymHoLAHiUnMbJcdc6xtAUlDCq7x2fb99sUFc="
     # WOO_UID for travis
     - secure: "jXqnDaiGjCUUAc4r865NkgbXCPfqRxrfWP+lKtAG7Lc/U6sTQBIJww1bsvfizFOdiEAgAzk4OAgE4z6djf7WbhHzyQKxGIieyLWUXoKhzAKjRYwidmKp6gBWOZSvcxURcYF2SpRAuKASpv/9lvZ3bNOlHu9LYQJ7r49AmCXJ98o="
+    # SNYK_TOKEN
+    - secure: "PmWGs/4SJH3bbB66E7eHLrrjKZ7mDnobAhQriRjpVjzGfKIpYqpiYGDF4rKVyYS9nkDgnZQaUErHFmmCcuZ603Rd4RikB/EKHPUtaWBT0IOGbrF0DqL27t/xOhkJVq9OUczLt2HVYoK1qFZXs/Q3bz5FvnNIrr4lp+oUE+gvSaY="
+
+install:
+  - npm install -g snyk
+  - npm install -g snyk-gradle-plugin
 
 stages:
-- name: build
-  if:  branch = master
-- name: test
-  if:  branch = master
-- name: scan
-  if:  branch = master AND NOT type = pull_request
-- name: snappah
-  if:  branch = master AND NOT type = pull_request
+  - name: build
+    if:  branch = master
+  - name: test
+    if:  branch = master
+  - name: scan
+    if:  branch = master AND NOT type = pull_request
```

Comment:
Here we need to remove the 'AND NOT type = pull_request' and on the line just below. This will enable the scans for  a PR.